### PR TITLE
Include prettier in the react-components package

### PR DESCRIPTION
**Pull request by bearni95**
Added `prettier` (2.8.5) dependency and a prettier npm command (`npm run prettier`) to the package.json for libs/react-components

### CODE REVIEW

File: common/config/rush/pnpm-lock.yaml [hunk 1]
```diff
@@ -708,6 +708,7 @@ importers:
       '@types/node': ^16.0.0
       '@types/react': ^18.0.28
       eslint: ^8.15.0
+      prettier: ^2.8.5
```
Comment (Randynamic) [inline common/config/rush/pnpm-lock.yaml:711]: If I'm not mistaken we already have a global prettier config. We do need to update that one to the latest version.

Comment (eileenmguo) [reply common/config/rush/pnpm-lock.yaml:711]: That was my understanding as well, but I think @alber70g knows best here. I think if the goal is to just fix formatting you can also run the `lint` script

Comment (bearni95) [reply common/config/rush/pnpm-lock.yaml:711]: Prettier runs upon committing with the `lint-staged` command, but I've found no way to run something like `npm run prettier` other than including it in the package

Comment (ash-vd) [reply common/config/rush/pnpm-lock.yaml:711]: You can run `rush lint` to lint everything in the monorepo, or run `rush lint -i .` to lint the project in the current working directory and the projects it depends on. It's also possible to for example run `rush lint -I @kadena/client` to specify a project.

We shouldn't add prettier as a dependency for each project, but rely on the one that's defined globally :).

Comment (bearni95) [reply common/config/rush/pnpm-lock.yaml:711]: You're right, there are other ways without messing with the package.json

Thanks!!

**Pull request closed without merging**